src/variable.cpp

```diff
@@ -4797,32 +4799,40 @@ double Variable::evaluate_boolean(char *str)
             delete[] str1;
             delete[] str2;
           }
+
         } else if (opprevious == LT) {
-          if (flag2) error->all(FLERR,"Invalid Boolean syntax in if command");
-          if (value1 < value2) argstack[nargstack].value = 1.0;
```

Comment:
@sjplimp this change introduces a bug (`$a < $b` is always false) that was flagged in extended regression testing. Will submit a PR with the fix (and a few others).

Reply:
oops - must have whacked that line for LT when refactoring - good catch